init.lua

```diff
@@ -18,44 +18,44 @@ minetest.register_chatcommand("nametag",{
 	func = function(user, param)
 		param = param:lower()
 		if param == "" then
-			minetest.after(1, minetest.show_formspec, user, "name-colours",
+			minetest.after(0, minetest.show_formspec, user, "name-colours",
 				"size[8.5,4.5]"..
-				"label[0.25,0.5;Choose a color:]"..
-				"button_exit[0.25,1.5;2.5,0.5;green;Green]"..
-				"button_exit[3,1.5;2.5,0.5;blue;Blue]"..
-				"button_exit[5.75,1.5;2.5,0.5;red;Red]"..
-				"button_exit[0.25,2.5;2.5,0.5;purple;Purple]"..
-				"button_exit[3,2.5;2.5,0.5;yellow;Yellow]"..
-				"button_exit[5.75,2.5;2.5,0.5;cyan;Cyan]"..
-				"button_exit[0.25,3.5;2.5,0.5;white;White]"..
-				"button_exit[3,3.5;2.5,0.5;orange;Orange]"..
+				"label[0.25,0.5;Choose a color:]" ..
+				"button_exit[0.25,1.5;2.5,0.5;green;Green]" ..
+				"button_exit[3,1.5;2.5,0.5;blue;Blue]" ..
+				"button_exit[5.75,1.5;2.5,0.5;red;Red]" ..
+				"button_exit[0.25,2.5;2.5,0.5;purple;Purple]" ..
+				"button_exit[3,2.5;2.5,0.5;yellow;Yellow]" ..
+				"button_exit[5.75,2.5;2.5,0.5;cyan;Cyan]" ..
+				"button_exit[0.25,3.5;2.5,0.5;white;White]" ..
+				"button_exit[3,3.5;2.5,0.5;orange;Orange]" ..
 				"button_exit[5.75,3.5;2.5,0.5;black;Black]"
 			)
 		elseif colors[param] then
 			local player = minetest.get_player_by_name(user)
-			player:set_nametag_attributes({
-				color = colors[param]
-			})
+			player:set_nametag_attributes({color = colors[param]})
+			player:set_attribute("nametag_color", minetest.serialize(colors[param]))
 			return true, "Your nametag color has been set to: " .. param
+		else
+			return false, "Incorrect Color! If you don't know what colours are avalible, just use /nametag without an option after it."
 		end
-		return false, "Invalid usage, see /help nametag"
 	end
 })
 
 -- Customized RGB-format color
 minetest.register_chatcommand("custom-nametag", {
 	params = "<red> <green> <blue>",
 	description = "Customizable-nametag color",
-	privs = {nametag = true},
+	privs = {shout = true},
```

Comment:
Why?

Reply:
Nametag privilege is never registered!

Reply:
:anger:  @Panquesito7

Reply:
@SmallJoker, AFAIK, you told me to remove it.
I forgot to change it to `shout` here.

Or, it can be added as an optional feature in this pull request.